app/src/main/java/com/nervousfish/nervousfish/LoginActivity.java

```diff
@@ -12,58 +13,50 @@
 @SuppressWarnings("PMD")
 public class LoginActivity extends Activity {
 
-    private static final String DEMO_EMAIL = "espresso@spoon.com";
-    private static final String DEMO_PASSWORD = "lemoncake";
+    private static final String DEMO_PASSWORD = "12345";
 
-    private EditText mEmail, mPassword;
+    private EditText mPassword;
     private View mError;
 
+    private boolean inDevelopment = true;
```

Comment:
Could you remove this? It is not really needed at this point in time... If you want to make sure we don't forget about the ability to login when no password is provided you can create an issue

The reason I don't want a variable like this is because if you want to be able to run the app in development or production mode this should be a global/environment variable and not a local (private) variable in a random class.